src/base/io/utilities/getDistributedModelFolder.m

```diff
@@ -0,0 +1,33 @@
+function  modelDir = getDistributedModelFolder(modelName) 
+% Identifies the folder a model is to be searched in.
+%
+% USAGE:
+%
+%    modelDir = getDistributedModelFolder(modelName) 
+%
+% INPUT:
+%    modelName:         The name of the model including the file extension
+%    
+%
+% OUTPUTS:
+%    modelDir:          The folder the model should eb located in.
+% 
+
+global CBTDIR
+if isempty(CBTDIR)
+    error('The Toolbox is not initialized. Cannot identfy the Model folder.\nPlease run initCobraToolbox() and try again');
```

Comment:
As you did in `getDistributedModel`, may you run here:
```Matlab
global ENV_VARS
ENV_VARS.printLevel = false;
initCobraToolbox;
ENV_VARS.printLevel = true;
```
instead of throwing an error?

Reply:
Nope. because this is called in initCobraToolbox... and this could otherwise cause an infinite loop if something goes wrong.

Reply:
ok 👍

Reply:
Hmmm. Thinking about it. If setting of CBTDIR fails we have bigger issues than a infinite loop :)

Reply:
Yes, true. In `changeCobraSolver`, these lines are present, which is also called in `initCobraToolbox`... Let's leave it for now.